pytorch_blade/src/compiler/mlir/converters/torch_mlir_op_filter.cpp

```diff
@@ -58,6 +58,7 @@ const std::unordered_set<std::string> &GetTorchMlirWhiteList() {
       "aten::glu",
       "aten::hardtanh",
       "aten::index_select",
+      "aten::layer_norm",
```

Comment:
also added `native_layer_norm` to white list?

Reply:
I think it can be added in another PR.

Reply:
done